docs/documentation-standards/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: doc
+title: Documentation Standards
+description: Documentation Standards for WP-CLI commands.
+category: References
+---
+
+We have set some Documentation Standards for WP-CLI commands.
```

Comment:
Also, it would be helpful to have some clarification of _why_ we have documentation standards, and when they should be applied.